PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/resources/META-INF/azure-sdk-reference-book.xml

```diff
@@ -1,5 +1,16 @@
 <idea-plugin>
     <resource-bundle>com.microsoft.azure.toolkit.operation.title.sdk</resource-bundle>
+    <extensions defaultExtensionNs="com.intellij">
+        <!-- dependencySupport: Notifies users that a plugin supporting a particular dependency is available and asks if they want to install it -->
+        <!-- COMMENTING OUT COMMAND LIBS FOR NOW:
+             The displayName is currently not getting used, resulting in dependency coordinates getting used.
+             See https://youtrack.jetbrains.com/issue/IDEA-275873 When it shows both the main lib and the command lib coordinates,
+             the resulting message is very dense, especially given how long the command coordinates are.
+             Since the command libs depend on the primary lib, we should be fine just specifying the main lib for now, giving a better message. -->
+        <dependencySupport displayName="Azure SDK client libraries" coordinate="com.azure:azure-core" kind="java"/>
+        <dependencySupport displayName="Azure Spring Boot client libraries" coordinate="com.azure.spring:azure-spring-boot" kind="java"/>
+        <dependencySupport displayName="Azure Resource Manager (ARM) client libraries" coordinate="com.azure:azure-core-management" kind="java"/>
```

Comment:
do we need to add `azure-core-management` here? Since it depends on `azure-core`, [refers](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-management/pom.xml#L65-L69)